### PR TITLE
Sed bugfix

### PR DESCRIPTION
**Pull request by adimako**
@cekees I refactored it to prevent memory leaking. I found some lines commented out because it would make drag nonlinear. Yet I 'd like these lines in, as they enable a smooth transition from packed situation to a suspended one. The line in the code is below

### CODE REVIEW

**Comment by adimako** [comment]
https://github.com/erdc-cm/proteus/blob/WT_optimisation/proteus/mprans/SedClosure.h#L81

File: proteus/mprans/SedClosure.h [hunk 1]
```diff
@@ -576,7 +576,7 @@ namespace proteus
 
     
 
-    inline double*  mIntFluid(  double sedF,
+    inline void  mIntFluid( double * mint2, double sedF,
```
Comment (cekees) [inline proteus/mprans/SedClosure.h]: On functions like this that "return" values by setting function arguments, there's an informal (and not consistently followed) convention to put the  inputs first and the outputs last.  mint2 is the output here, right?

Comment (adimako) [inline proteus/mprans/SedClosure.h]: Correct. . I will merge for now but I will update all of them later (I did the same in WaveTools.h)

Comment (adimako) [inline proteus/mprans/SedClosure.h]: Correct. . I will merge for now but I will update all of them later (I did the same in WaveTools.h)

**Comment by adimako** [comment]
@cekees I have not merged this one yet. I was about to but then I thought I would add a solitary wave theory to WaveTools. 
I used Eleni's formulation and I created the unit tests by copying / pasting the formulation in OpenFOAM source code, where we know that it works as we ahve tested it. It passed all tests, so unless we there are any objections, I say we merge, if the tests pass

**Comment by cekees** [comment]
+1

**Pull request merged**